cmd/orb.go

```diff
@@ -44,6 +44,7 @@ type orbOptions struct {
 	listUncertified bool
 	listJSON        bool
 	listDetails     bool
+	selectPrivate   bool
```

Comment:
Any reason this couldn't be named `private` or `isPrivate`? Not sure what the convention is.

Reply:
`private` makes sense. The other ones had a verb in front so `isPrivate` didn't seem right